prism/core.py

```diff
@@ -108,7 +108,12 @@ def get_thumb_filename(file_name, cmd, options):
             params.append(f'{k}__{value}')
     param_string = '--'.join(params)
 
-    extension = '.%s' % options['out_format']
+    out_format = options['out_format']
+    if out_format == "":
+        extension = ".png"
```

Comment:
extension = "" 
Didn't work I guess :(

Reply:
Sorry @cemonel, I couldn't understand the comment. Can you give an example please?

Reply:
We could try to add extension as "". So we will have only cid and the options in the thumbnail image name.

Reply:
I tried this, it didn't work, we can go with this approach.

Reply:
> If we save without any extension, Browsers fail to show in the tab and try to download. Because of this, I fallback to .png.
Yes yes. 
I just wanted to test if the `.` at the and is breaking something or not as we talked.